UsefulStuff.Core/Features/ImprovedTransformOperations.cs

```diff
@@ -551,60 +565,75 @@ private void UpdateButtonsVisibility()
 
             private void CopyTransform()
             {
-                GuideObject source = _hashSelectObject.First();
-                _savedPosition = source.changeAmount.pos;
-                _savedRotation = source.changeAmount.rot;
-                _savedScale = source.changeAmount.scale;
-                _clipboardEmpty = false;
-                UpdateButtonsVisibility();
+                if (_hashSelectObject.Count == 1)
+                {
+                    GuideObject source = _hashSelectObject.First();
+                    _savedPosition = source.changeAmount.pos;
+                    _savedRotation = source.changeAmount.rot;
+                    _savedScale = source.changeAmount.scale;
+                    _clipboardEmpty = false;
+                    UpdateButtonsVisibility();
+                }
             }
 
+            // Without this the compiler will complain
+            // `cannot convert from 'method group' to 'UnityAction'`
+            // when adding this function to a listener
             private void PasteTransform()
+            {
+                PasteTransform(true, true, true);
+            }
+
+            private void PasteTransform(bool pastePos = false, bool pasteRot = false, bool pasteScale = false)
```

Comment:
It would be better to not have default values here I think, or at least have the default values all be true like in `PasteTransform()` above. As it is now it can be unclear which method will be used, which is a problem because they have the completely opposite effects.

Reply:
True, I had the readability of calling the function in mind, but didn't think about the ambiguity. Removed the default values to solve that.